provisioner/template.go

```diff
@@ -124,6 +127,13 @@ func renderTemplate(context *templateContext, file string) (string, error) {
 		"awsValidID":                            awsValidID,
 		"indent":                                sprig.GenericFuncMap()["indent"],
 		"dict":                                  dict,
+		"scaleQuantity":                         scaleQuantity,
+		"instanceTypeCPU": func(instanceType string) (string, error) {
+			return instanceTypeCPU(context, instanceType)
+		},
+		"instanceTypeMemory": func(instanceType string) (string, error) {
+			return instanceTypeMemory(context, instanceType)
+		},
```

Comment:
Maybe add `Quantity` suffix to highlight the format, i.e. `instanceTypeCPUQuantity` and `instanceTypeMemoryQuantity`